openshift/amunInspectionWorkflow-template.yaml

```diff
@@ -0,0 +1,169 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: amun-inspection-workflow
+  annotations:
+    description: This is Amun configuration template
+    openshift.io/display-name: "Amun core components"
+    version: 0.5.1
+    tags: amun,argo,thoth,ai-stacks,aistacks,machinelearning,workflows
+    template.openshift.io/documentation-url: https://github.com/thoth-station/
+    template.openshift.io/long-description: This template defines resources needed to deploy Amun to OpenShift.
+    template.openshift.io/provider-display-name: Red Hat, Inc.
+    thoth-station.ninja/template-version: 0.5.1
+  labels:
+    template: amun-inspection-workflow
+    app: amun
+
+objects:
+- apiVersion: argoproj.io/v1alpha1
+  kind: Workflow
+  metadata:
+    generateName: "${AMUN_INSPECTION_ID}"
```

Comment:
This is to identify the workflow instance, correct?

Reply:
Correct.